apps/web/app/api/code/submission/route.ts

```diff
@@ -0,0 +1,124 @@
+import { NextRequest, NextResponse } from "next/server";
+import db from "@repo/db/client";
+
+const getInputString = (args: string[], funcName: string) => {
+  return `\n
```

Comment:
Should this take language Id as input and be different based on the language?

Reply:
Yes it will, have to work on it, it's pending